speech/unit_tests/test_sample.py

```diff
@@ -42,6 +42,39 @@ def test_content_and_source_uri(self):
             self._make_one(content='awefawagaeragere',
                            source_uri=self.AUDIO_SOURCE_URI)
 
+    def test_content_to_stream(self):
+        from google.cloud.speech.encoding import Encoding
+
+        content = 'testing 1 2 3 4'
+        sample = self._make_one(content=content,
+                                encoding=Encoding.FLAC,
+                                sample_rate=self.SAMPLE_RATE)
+
+        self.assertIsNotNone(getattr(sample.stream, 'read', None))
+
+    def test_content_is_file_like(self):
+        from io import BytesIO
+        from google.cloud.speech.encoding import Encoding
+
+        test_bytes = b'testing 1 2 3 4'
+        content = BytesIO(test_bytes)
+        sample = self._make_one(content=content,
+                                encoding=Encoding.FLAC,
+                                sample_rate=self.SAMPLE_RATE)
+        self.assertEqual(sample.content, test_bytes)
+
+    def test_bytes_converts_to_file_like_object(self):
```

Comment:
Now it just returns `None` if it's not a file like object.
https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2850/commits/bd7130537a3ea852593966997245c6d9dd0235a3#diff-37554bc08e0770c52c2125924d18ae12R126